### PR TITLE
Standard page models

### PR DESCRIPTION
**Pull request by akmiller01**
Standard page template, abstract content page model with `fixed_page_type` choice field anticipating two special uses of this page for the `privacy page` and the `terms and conditions page`. Re-usable otherwise.

### CODE REVIEW

File: iati/iati/templates/base.html [hunk 1]
```diff
@@ -148,8 +148,8 @@
 					<h4 class="footer__heading">Useful links</h4>
 					<ul class="footer__list">
 						<li><a href="{% default_page_url 'contact' %}">Contact</a></li>
-						<li><a href="{% default_page_url %}">Terms and conditions</a></li>
-						<li><a href="https://www.aidtransparency.net/privacy-policy">Privacy policy</a></li>
+						<li><a href="{% standard_page_url 't' %}">Terms and conditions</a></li>
```
Comment (allthatilk) [inline iati/iati/templates/base.html]: See comment about more descriptive key name

File: iati/iati/templates/base.html [hunk 2]
```diff
@@ -148,8 +148,8 @@
...
+						<li><a href="{% standard_page_url 'p' %}">Privacy policy</a></li>
```
Comment (allthatilk) [inline iati/iati/templates/base.html]: See comment about more descriptive key name

File: iati/tests/home_functional_tests.py [hunk 3]
```diff
@@ -1 +1,90 @@
 """A module of functional tests for the home page."""
+import os
+import pytest
+from django.utils.text import slugify
+
+
+PRIVACY_PAGE = {
+    'page_type': 'Standard page',
+    'fixed_page_type': 'p',
```
Comment (allthatilk) [inline iati/tests/home_functional_tests.py]: See comment about more descriptive key name

File: iati/tests/home_functional_tests.py [hunk 4]
```diff
@@ -1 +1,90 @@
...
+    'title': 'Privacy page',
+    'heading': 'Privacy page header',
+    'base_link_text': 'Privacy policy'
+}
+TERMS_PAGE = {
+    'page_type': 'Standard page',
+    'fixed_page_type': 't',
```
Comment (allthatilk) [inline iati/tests/home_functional_tests.py]: See comment about more descriptive key name

File: iati/tests/home_functional_tests.py [hunk 5]
```diff
@@ -1 +1,90 @@
...
+    tab = admin_browser.find_by_text(tab_name)
+    _ = tab[0].__dict__['_element'].location_once_scrolled_into_view
+    tab[0].click()
+    elem = admin_browser.find_by_css("[name='{}']".format(cms_field))
+    _ = elem[0].__dict__['_element'].location_once_scrolled_into_view
+    admin_browser.fill(cms_field, cms_content)
+
+
+def publish_page(admin_browser):
+    """Publish page created in the CMS."""
```
Comment (allthatilk) [inline iati/tests/home_functional_tests.py]: Can you add a note in this docstring that this is a duplicate function?

File: iati/tests/home_functional_tests.py [hunk 6]
```diff
@@ -1 +1,90 @@
...
+    admin_browser.find_by_xpath('//div[@class="dropdown-toggle icon icon-arrow-up"]').click()
+    admin_browser.find_by_text('Publish').click()
+
+
+def create_standard_home_page(admin_browser, page_type, fixed_page_type, page_title):
+    """Create a child page in the CMS.
+
+    Args:
+        page_type (str): The verbose name of the page model type you want to click on.
+        fixed_page_type (str): The value of the fixed page dropdown (e.g. "p" for "privacy")
```
Comment (allthatilk) [inline iati/tests/home_functional_tests.py]: This will need updating once key names are updated

**Comment by akmiller01** [comment]
Hey @allthatilk Updated all the key names and added the note you requested.

**Pull request merged**